.buildkite/pipeline.full.yml

```diff
@@ -83,7 +81,7 @@ steps:
     concurrency: 9
     concurrency_group: 'browserstack-app'
 
-  - label: ':android: Android 4.4 NDK r16 end-to-end tests'
+  - label: ':android: Android 4.4 NDK r16 end-to-end tests - batch 1'
```

Comment:
I'd propose a logical way of splitting scenarios would be to run all the NDK/ANR scenarios in one job, and everything else in another. There are roughly ~150 scenarios in total and this approximately splits them in half, _and_ logically orders them in a directory structure.

Reply:
Sounds good to me - could you raise an internal ticket and send it my way?  If it can just makes the separation clear so there's no margin for error and we'll make it happen in a forthcoming sprint.

Reply:
Raised an internal ticket